### ISSUE TITLE
Pipelines providers google_manifold_pipeline 

### ISSUE THREAD

**Issue by derevyan** [open-webui/pipelines#154]
Pipeline : https://github.com/open-webui/pipelines/blob/main/examples/pipelines/providers/google_manifold_pipeline.py

After last updates return: Could not fetch models from Google, please update the API Key in the valves.
Works great before, all API keys "GOOGLE_API_KEY" set correctly. During request to /models i can see 200 return result from Google. ("GET /models HTTP/1.1" 200 OK)
Anthropic pipeline works great.

v0.3.7
[(latest)](https://github.com/open-webui/open-webui/releases/tag/v0.3.7)

Today Docker image from: ghcr.io/open-webui/pipelines:main

Thanks a lot for great product and code guys.

**Comment by justinh-rahb** [comment]
Confirming this issue but I'm at a loss as to what could have caused it... the Pipelines server code has not had any major changes recently, and neither has the manifold code. My attempts at fixing this have only lead me to even more nonsensical errors, so I'm forced to conclude at this point that possible Google changed the library or the API 🤷‍♂️

**Comment by pressdarling** [comment]
I've been getting this with both Google and Anthropic - @justinh-rahb thanks for both of those scripts, even if they aren't quite working at the moment! For now, the open-webui Functions seems to work, and I've removed the pipelines for the time being, but I get errors with custom models where I've set a system message on both providers. I'll try to remember to raise an issue over there if I find time to investigate further.

I'm also using Cohere (function) and Groq (Pipeline manifold), both are fine. Can share logs if there's anything useful, was poking around yesterday and saw nothing obvious to me.
